tests/cases/fourslash/refactorConvertToGetAccessAndSetAccess35.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path='fourslash.ts' />
+
+//// class A {
+////     /*a*/public/*b*/ /*c*/a/*d*/ = () => {
+////         /*e*/return/*f*/ /*g*/1/*h*/;   
+////     }
+////     /*i*/b/*j*/: /*k*/number/*l*/ = /*m*/1/*n*/
+//// };
+
+goTo.select("a", "b");
+verify.refactorAvailable("Generate 'get' and 'set' accessors");
+
+goTo.select("c", "d");
+verify.refactorAvailable("Generate 'get' and 'set' accessors");
+
+goTo.select("e", "f");
+verify.not.refactorAvailable("Generate 'get' and 'set' accessors");
```

Comment:
I would prefer just `verify.not.refactorAvailable();`, otherwise this test will trivially pass if we ever change the description.